rest-api/src/main/java/gov/cms/qpp/conversion/api/model/Metadata.java

```diff
@@ -255,6 +255,26 @@ public void setOverallStatus(Boolean overallStatus) {
 		this.overallStatus = overallStatus;
 	}
 
+	/**
+	 * Whether a submission was part of a test
+	 *
+	 * @return Test or valid
+	 */
+	@DoNotEncrypt
+	@DynamoDBAttribute(attributeName = "Test")
+	public Boolean getTest() {
```

Comment:
I would like to verify that a signature mismatch doesn't occur when retrieving a `Metadata` that doesn't already contain this `Test` attribute.  Has this been verified?

The framework we use verifies a signature of the `Metadata` in DynamoDB whenever we retrieve it.  This prevents us from modifying this `Metadata` outside of our application (such as through the AWS Console directly).  If there is a signature mismatch, an exception is thrown.  Therefore, I would like to verify that this signature mismatch doesn't occur when adding this new attribute and retrieving older `Metadata` that doesn't already have this attribute.